app/models/project.server.ts

```diff
@@ -111,55 +111,56 @@ export async function searchProjects({
   take = 50,
 }: SearchProjectsInput) {
   let where = Prisma.sql`WHERE p.id IS NOT NULL`;
+  let having = Prisma.empty;
   if (search && search !== "") {
     search === "myProposals"
       ? (where = Prisma.sql`WHERE pm."profileId" = ${profileId}`)
       : (where = Prisma.sql`WHERE "tsColumn" @@ websearch_to_tsquery('english', ${search})`);
   }
 
-  const statuses = typeof status === "string" ? [status] : []
-  if (status) {
-    where = Prisma.sql`${where} AND p.status IN (${Prisma.join(statuses)}) `
+  if (status.length > 0) {
```

Comment:
it was possible in blitz, but not anymore in remix, because to send them we are using [URLSearchParams.getAll](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/getAll) that returns a string array.